Personalizing your Skywallet.md

```diff
@@ -0,0 +1,110 @@
+# Personalizing your Skywallet
+
+After completing the initialization of your Skywallet, you can start to personalize your Skywallet using the Hardware Configuration window in the desktop wallet.
+
+Upon configuring your Skywallet using your desktop wallet, the Skywallet will be listed in the Wallet list.
+
+The list can be viewed when you open your Desktop wallet.
+
+<Screenshot of the list of wallets displayed when the desktop wallet is opened. One of the wallets in the list needs to be a Skywallet>
+
+To personalize your Skywallet, you have to click on the **"Hardware Wallet"** button.
+
+<Screenshot of the desktop wallet highlighting the Hardware wallet button>
+
+Once you click on this button, the following window will appear:
+
+<Screenshot of the Hardware configuration window with the options available when the Skywallet is used after configuration for the first time>
+
+During your first time use of the Skywallet, you will find these two warnings associated with **"Creating a Backup"** and **"Setting a Pin"** being displayed in the Desktop wallet window.
+
+Both of these warnings are to be dealt with seriously or it may compromise the security of your Skywallet.
+
+In order to complete these two tasks, you can use the options in the Hardware Configuration window
+
+Following are the options available in this window:
+
+**Create a backup**
+
+**Create PIN code**
+
+**Wipe the device**
+
+## Create a backup
+
+The Seed of your Skywallet is a key component. Hence, once the initialization is complete it is important to backup this Seed and store it in a secure location.
+
+To backup the Seed of your skywallet, click on the button **"Create a backup"**.
+
+<Screenshot of the hardware configuration window with an arrow pointing towards the create a backup button or highlighting the button>
+
+Once you click the button, a warning message(?) will be displayed.
+
+After accepting the same the Skywallet will start to display the words in your Seed one by one.
+
+<The warning message needs to be more clearly mentioned and possibly with the screenshot of the warning message>
+<Picture of the Skywallet display screen showing different words in the Seed> <it would be ideal to put at least two pictures>
+
+**You need to note down each of these words and store it in a secure location.** 
+
+**Note - Make sure that it is retrievable because if anything happens to your device, it will be impossible to retrieve your coins without the Seed.**
+
+After completing the display of the words in the Seed, the screen displays ***all the words in the seed again***, so that you can verify your entries.
+
+<Needs to add a comment on using the button confirm seed after backing up the seed>
```

Comment:
What  does this comment mean?

Reply:
We have included the steps involved while using the "Create Backup" button. However, after completing the process of the backup, the button will be replaced by "confirm seed" button.
But explaining this button might make the manual too long. So I was having doubt whether to include it or just add a comment that the confirm seed button will be appearing after the first time use of create backup button. That is why I have put that comment.